### PR TITLE
Separate personal description from the main one

### PR DESCRIPTION
**Pull request by irhel**

### CODE REVIEW

File: src/extractors/desktop.js [hunk 1]
```diff
@@ -66,17 +66,28 @@ exports.extractOrganicResults = ($) => {
...
+        let mainDescription = null;
+        let personalDescription = null;
+        if ($(el).find('[data-content-feature="1"]').children().length === 2) {
+            mainDescription = $(el).find('[data-content-feature="1"]').children().eq(1);
+            personalDescription = $(el).find('[data-content-feature="1"]').children().eq(0)
+                .text()
+                .trim();
+        } else {
+            mainDescription = $(el).find('[data-content-feature="1"]');
+        }
```
Comment (metalwarrior665) [inline src/extractors/desktop.js:78]: We will definitely want to split it into separate fields for name, company etc.

**Review by metalwarrior665 — CHANGES_REQUESTED** [review]

File: src/extractors/desktop.js [hunk 2]
```diff
@@ -66,17 +66,29 @@ exports.extractOrganicResults = ($) => {
...
+            ...extractDescriptionAndDate(mainDescription.text()),
             emphasizedKeywords: $(el).find('.VwiC3b em, .VwiC3b b').map((_i, element) => $(element).text().trim()).toArray(),
             siteLinks,
             productInfo,
         };
-
+        if (personalDescription) {
+            [searchResult.name] = searchResult.title.split('-').map((field) => field.trim());
+            [searchResult.location, searchResult.jobTitle, searchResult.companyName] = personalDescription.split('·').map((field) => field.trim());
+        }
```
Comment (metalwarrior665) [inline src/extractors/desktop.js:91]: Let's put it into `personalInfo` object. 

Also, check the code for `productInfo`, I'm afraid the selectors might be the same for products as for names

**Review by metalwarrior665 — CHANGES_REQUESTED** [review]
Do you have some test runs?

File: src/extractors/desktop.js (hunk 2 above)
Comment (irhel) [reply src/extractors/desktop.js:91]: Tested it locally. I am not getting the fields by selectors but by the order in which they appear.

**Review by metalwarrior665 — APPROVED** [review]
Ok ,let's test a few cases

**Pull request merged**